languages/js/src/errors.ts

```diff
@@ -35,11 +35,11 @@ export class DuplicateInstanceRegistrationError extends PolarError {
 }
 
 export class InlineQueryFailedError extends PolarError {
-  constructor(file?: string) {
+  constructor(source: string, file?: string) {
     if (file !== undefined) {
-      super(`Inline query failed while loading '${file}'.`);
+      super(`Inline query failed while loading '${file}', ${source}.`);
     } else {
-      super('Inline query failed.');
+      super(`Inline query failed, ${source}.`);
```

Comment:
I think these commas would be better as colons, maybe with the source indented on the following line. Something like:
```
Inline query failed while loading file policies/production.polar:
  ?= 1 = 2;
```